tests/functional/spec/features/routing/number.spec.js

```diff
@@ -0,0 +1,188 @@
+const helpers = require('../../../helpers');
+
+describe('Feature: Routing on a Number', function () {
+  describe('Equals', function () {
+    describe('Given I start number routing equals survey', function () {
+
+      var NumberQuestionPage = require('../../../pages/features/routing/number/equals/number-question.page');
+      var CorrectAnswerPage = require('../../../pages/features/routing/number/equals/correct-answer.page');
+      var IncorrectAnswerPage = require('../../../pages/features/routing/number/equals/incorrect-answer.page');
+
+      before(function () {
+        return helpers.openQuestionnaire('test_routing_number_equals.json');
+      });
+
+      it('When i enter the correct number, Then i should be routed to the correct page', function () {
```

Comment:
Minor thing shouldn't the i's be upper case? Just to match how we do it across the the rest of the specs